.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-* @openfga/backend
+* @openfga/backend @openfga/dx
```

Comment:
I'll revert this prior to merge so that I can still easily get any changes into the feat/stronger-consistency-options branch if needed